### ISSUE TITLE
"Platform" parameter values on deliver is not up to date

### ISSUE THREAD

**Issue by zackdotcomputer** [fastlane/docs#1073]
The docs for deliver still list the valid platform values as "mac" and "ios" and whether these should be symbols or strings is not defined. However, in usage the tool requires a string and says the valid values are "ios, appletvos, or osx". The docs should be updated to reflect this.

**Comment by zackdotcomputer** [comment]
(Sorry to make an issue not a PR - I intend to tackle this but don't have the time to fork and fix at the moment so putting this here in case someone gets to it first)

**Comment by ainame** [comment]
Good point. However, we generate docs for "actions" from the corresponding code so that we don't have the source of the document here. I'll transfer this to fastlane/fastlane.
